components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -2527,6 +2527,54 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+######################################################
+# The get usage billing resource APIs
+######################################################
+  '/subscriptions/{subscriptionId}/usage':
+    get:
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" https://127.0.0.1:9443/api/am/store/v1.0/subscriptions/64eca60b-2e55-4c38-8603-e9e6bad7d809/usage"
+      x-wso2-request: |
+        GET https://127.0.0.1:9443/api/am/store/v1.0/subscriptions/64eca60b-2e55-4c38-8603-e9e6bad7d809/usage
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+      x-wso2-response: "HTTP/1.1 200 OK\nContent-Type: application/json\n\n{\n   \"object\": \"invoice\",\n   \"account_country\": \"US\",\n   \"account_name\": \"test.com\",\n   \"amount_due\": \"0\",\n   \"amount_paid\": \"0\"\n  \"amount_remaining\": \"0\"\n}"
+      security:
+        - OAuth2Security:
+          - apim:sub_manage
+      summary: Get details of a pending invoice for a monetized subscription with meterd billing.
```

Comment:
```suggestion
      summary: Get details of a pending invoice for a monetized subscription with metered billing.
```

Reply:
Done in : https://github.com/wso2/carbon-apimgt/pull/6961/commits/0832832ce6d6c5e07bb85ed4d0846dafa034f88b